src/lib/components/Filter/Filter.js

```diff
@@ -7,6 +7,8 @@ import {
   buildFilterState,
   clearFilterState,
   lngLat,
+  mergeFilters,
+  buildFiltersMap
 } from 'gisida';
```

Comment:
@ciremusyoka What version / branch of gisida does this branch required?

Reply:
@cKellyDesign gisida branch is map-refactor which have also created a pull request (https://github.com/onaio/gisida/pull/387)  for.

Reply:
@ciremusyoka looks like the Gisida branch was merged, should we merge this now as well?